jbmc/src/java_bytecode/select_pointer_type.cpp

```diff
@@ -222,3 +222,12 @@ select_pointer_typet::get_recursively_instantiated_type(
   visited.erase(parameter_name);
   return inst_val;
 }
+
+optionalt<std::set<symbol_typet>>
+select_pointer_typet::get_parameter_alternative_types(
+  const irep_idt &function_name,
+  const irep_idt &parameter_name,
+  const namespacet &ns) const
+{
+  return {};
```

Comment:
I don't seen any other implementation of this method, so I don't really see its purpose.

Reply:
it's currently only implemented in smart pointer selection, this will later be ported to JBMC